server/graphql-schema.js

```diff
@@ -1132,10 +1074,11 @@ const graphqlSchema = gql`
         """
         deleteTestPlanRun(userId: ID!): PopulatedData!
         """
-        Update the report status. Remember that all conflicts must be resolved
-        when setting the status to CANDIDATE. Only available to admins.
+        Update the approvedAt date. This must be set before a TestPlanReport can
+        be advanced to CANDIDATE. All conflicts must also be resolved.
+        Only available to admins.
         """
-        updateStatus(status: TestPlanReportStatus!): PopulatedData!
+        updateApprovedAt: PopulatedData!
```

Comment:
Could I suggest calling this just "approve"?

Or maybe "markAsFinal" possibly.

Reply:
Agreed and updated to make this clearer.

Related to this, there was a discussion around the column property's name at https://github.com/w3c/aria-at-app/issues/632#issuecomment-1652681110

> Option B: Change data element to "markedFinalAt"

I've updated the `approvedAt` property to be `markedFinalAt` instead for consistency across the client and server + database implementation with this operation

Reply:
Great change! Thanks!